ArduPlane/tailsitter.cpp

```diff
@@ -150,6 +150,14 @@ static const struct AP_Param::defaults_table_struct defaults_table_tailsitter[]
     { "RUDD_DT_GAIN",      10 },
     { "Q_TRANSITION_MS",   2000 },
     { "Q_TRANS_DECEL",    6 },
+    { "Q_ACCEL_P_MAX.    30000},
+    { "Q_ACCEL_R_MAX.    30000},
+    { "Q_ACCEL_Y_MAX.    30000},
```

Comment:
```suggestion
    { "Q_A_ACCEL_P_MAX.    30000},
    { "Q_A_ACCEL_R_MAX.    30000},
    { "Q_A_ACCEL_Y_MAX.    30000},
```
For refence the current values are 110000, 110000 and 27000

Reply:
duh